### DISCUSSION TITLE
How to access "Priority Email Support" from Harman

### DISCUSSION THREAD

**Discussion by Work1SaurGitHub** [airsdk/Adobe-Runtime-Support#629] [Q&A]
Hello

We bought a couple of enterprise level licenses this month. I am wondering how to access the "priority email support". I have not been getting any response from the email address on the Harman site.  If how to do this is documented somewhere I would be grateful if someone could post a link.

**Comment by ajwfrost** [comment]
Hi - I just looked in the inbox but couldn't see a message from you (although there are a lot of messages in there, which perhaps is part of the problem... we have another business line with Adobe that is using the same inbox and is seeing a lot of traffic so the guys may be struggling to pick up actual requests for AIR support!)

Feel free to email me (andrew.frost at harman.com) and I'll forward it on to the appropriate team member..

thanks

**Comment by Work1SaurGitHub** [comment]
Thank you. I will do that.
